File: src/main/java/no/avexis/dropwizard/mail/SSLMailerClient.java
package no.avexis.dropwizard.mail;

import javax.mail.Session;

public class SSLMailerClient {
    Session session;

    SSLMailerClient(Session session) {
        this.session = session;

    }

    public void send(final String to, String subject, String text, boolean html) {
        EmailUtils.sendEmail(session, to, subject, text);
    }

}

File: src/main/java/no/avexis/dropwizard/mail/SSLMailerClient.kt
package no.avexis.dropwizard.mail

import com.fasterxml.jackson.annotation.JsonProperty
import com.google.common.base.Preconditions
import io.dropwizard.setup.Environment
import org.simplejavamail.email.Email
import org.simplejavamail.email.EmailBuilder
import org.simplejavamail.mailer.Mailer
import org.simplejavamail.mailer.config.TransportStrategy
import java.net.Authenticator
import java.net.PasswordAuthentication
import java.util.*
import javax.mail.Session


class SSLMailerClient(val session: Session, private val fromName: String, private val fromAddress: String) {
    fun send() {

    }
}
